blazor/dropdown-list/filtering.md

```diff
@@ -7,103 +7,116 @@ control: DropDown List
-
-<SfDropDownList TValue="string" @ref="ddlObj" TItem="Country" Placeholder="e.g. Australia" DataSource="@Countries" AllowFiltering="true">
-    <DropDownListFieldSettings Text="Name" Value="Code"></DropDownListFieldSettings>
-    <DropDownListEvents TValue="string" TItem="Country" Filtering="OnFilter"></DropDownListEvents>
-</SfDropDownList>
-
-
-@code {
-
-    SfDropDownList<string, Country> ddlObj { get; set; }
-
-    public class Country
-    {
-        public string Name { get; set; }
-
-        public string Code { get; set; }
-    }
-
-    List<Country> Countries = new List<Country>
-    {
-        new Country() { Name = "Australia", Code = "AU" },
-        new Country() { Name = "Bermuda", Code = "BM" },
-        new Country() { Name = "Canada", Code = "CA" },
-        new Country() { Name = "Cameroon", Code = "CM" },
-        new Country() { Name = "Denmark", Code = "DK" }
-    };
-
-    List<Country> CountriesFiltered = new List<Country>
-    {
-        new Country() { Name = "France", Code = "FR" },
-        new Country() { Name = "Finland", Code = "FI" },
-        new Country() { Name = "Germany", Code = "DE" },
-        new Country() { Name = "Greenland", Code = "GL" }
-    };
-
-    private async Task OnFilter(FilteringEventArgs args)
-    {
-        args.PreventDefaultAction = true;
-        var query = new Query().Where(new WhereFilter() { Field = "Name", Operator = "contains", value = args.Text, IgnoreCase = true });
-
-        query = !string.IsNullOrEmpty(args.Text) ? query : new Query();
-
-        await ddlObj.FilterAsync(CountriesFiltered, query);
-    }
-}
-```
\ No newline at end of file
+## Local data
+
+In the below code, demonstrated the filtering functionality with local data in the DropDownList component.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/local-data.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with local data filtering](./images/filtering/blazor_dropdown_local-data.png)
+
+## Remote data
+
+In the below code, demonstrated the filtering functionality with [ODataAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odata-adaptor) in the DropDownList component with help of [Query](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.Query.html) property.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/remote-data.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with Remote Data filtering](./images/filtering/blazor_dropdown_remote-data.png)
+
+## Filter type
+
+You can use [FilterType](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_FilterType) property to specify on which filter type, the component needs to be considered on search action. The available `FilterType` and its supported data types are:
+
+FilterType     | Description
+------------ | -------------
+  StartsWith       | Checks whether a value begins with the specified value.
```

Comment:
Link the "StratWith", EndWith, and Contains API link - https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.FilterType.html#Syncfusion_Blazor_DropDowns_FilterType_Contains